pyproject.toml

```diff
@@ -14,6 +14,7 @@ exclude = ["afids_utils/tests"]
 python = ">=3.8, <3.12"
 attrs = "^23.1.0"
 numpy = "^1.24.3"
+more-itertools = "^10.1.0"
```

Comment:
I might have missed this, but I thought this was only used in testing. In which case this can be moved to the dev group.

Reply:
Ah, good catch.